.github/workflows/request-staging-deployment.yml

```diff
@@ -0,0 +1,30 @@
+name: Request Staging Deployment
+
+on:
+    push:
+      branches-ignore:
+        master
+    pull_request:
```

Comment:
Yeah I noticed that as well 🤔 I _think_ we can remove the `pull_request` trigger since theoretically the action will have run as soon as you push your branch up (before creating a PR) and your deployment should show up in the PR timeline. I'll test with another branch to make sure

Reply:
Actually I think we want to remove the `push` trigger